Targets/AppUIKit/Sources/verifier/VerifyEAView.swift

```diff
@@ -0,0 +1,57 @@
+import SwiftUI
+import SpruceIDMobileSdkRs
+
+struct VerifyEA: Hashable {}
+
+struct VerifyEAView: View {
+
+    @State var stepOneValue: String?
+    @State var success: Bool?
+
+    @Binding var path: NavigationPath
+
+    var body: some View {
+        if success == nil {
+            if stepOneValue == nil {
+                ScanningComponent(
+                    path: $path,
+                    scanningParams: Scanning(
+                        subtitle: "Scan the front of your\nemployment authorization",
```

Comment:
same as before for breaklines

Reply:
this is just to make it equal to the designs